code/modules/mob_spawn/ghost_roles/spider_roles.dm

```diff
@@ -93,7 +96,7 @@
 	return ..()
 
 /obj/effect/mob_spawn/ghost_role/spider/process(seconds_per_tick)
-	amount_grown += rand(0, 1) * seconds_per_tick
+	amount_grown += rand(0, 10) * seconds_per_tick
```

Comment:
This introduces a large degree of variance, are you sure that is what you want?

Reply:
It doesn't gain nothing or 10, it gains anywhere between 0 and 10.
It could gain 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, or 10.
This makes the time it will take much more random than it was previously, it could still take as long as it did previously.

Reply:
MM im not sure if this randomness is a good thing or a bad thing what do you think

Reply:
eeh ill change it to 10-20